assets/src/dashboard/app/views/editorSettings/publisherLogo/index.js

```diff
@@ -66,39 +71,157 @@ function PublisherLogoSettings({
   publisherLogos,
   uploadError,
 }) {
+  const { isRTL } = useConfig();
+
+  const containerRef = useRef();
+  const gridRef = useRef();
+  const itemRefs = useRef({});
+
+  const [activePublisherLogo, _setActivePublisherLogoId] = useState(null);
+  const [indexRemoved, setIndexRemoved] = useState(null);
+
+  const publisherLogosById = useMemo(() => publisherLogos.map(({ id }) => id), [
+    publisherLogos,
+  ]);
+
+  const [publisherLogoCount, setPublisherLogoCount] = useState(
+    publisherLogosById.length
+  );
```

Comment:
Since this is just stored as a state to see how many previous logos there were let's change this to a `useRef`
`const publisherLogoCount = useRef(publisherLogosById.length);`

That way we don't fire off additional renders.